physical/raft/fsm.go

```diff
@@ -154,10 +154,16 @@ func (f *FSM) openDBFile(dbPath string) error {
 		return errors.New("can not open empty filename")
 	}
 
-	boltDB, err := bolt.Open(dbPath, 0o666, &bolt.Options{Timeout: 1 * time.Second})
+	start := time.Now()
+	boltDB, err := bolt.Open(dbPath, 0o666, &bolt.Options{
+		Timeout:        1 * time.Second,
+		FreelistType:   bolt.FreelistMapType,
+		NoFreelistSync: true,
```

Comment:
Could also we add an option to go back to the old settings?

Reply:
Is it safe?  https://github.com/etcd-io/bbolt/pull/1#issuecomment-309948066

Reply:
I can pretty easily run a few tests for converting both directions and see what the results are.

Reply:
I think the safety was added in https://github.com/etcd-io/bbolt/pull/6/

Reply:
Ah nice